app/services/catalog/share_info.rb

```diff
@@ -10,10 +10,13 @@ def initialize(options)
 
     def process
       group_permissions = {}
+      uuids = []
       @object.access_control_entries.each do |ace|
         group_permissions[ace.group_uuid] = ace.permissions.map(&:name)
+        uuids << ace.group_uuid
```

Comment:
Should you use a `collect` to gather the `uuids`?

Reply:
@gmcculloug I am trying to use the same loop, otherwise I would have to do the collect in a separate loop.

Reply:
You still can.  Collect returns an array of the objects from the last line in the block.

This will give you the same result:
```ruby
group_permissions = {}
uuids = @object.access_control_entries.collect do |ace|
  group_permissions[ace.group_uuid] = ace.permissions.map(&:name)
  ace.group_uuid
end
```